tests/Microsoft.Identity.Test.Unit/WwwAuthenticateParametersTests.cs

```diff
@@ -56,6 +58,77 @@ public void CreateWwwAuthenticateResponse(string resource, string authorizationU
             Assert.IsNull(authParams.Error);
         }
 
+        [TestMethod]
+        [DataRow("WLID1.0", "realm=WindowsLive, policy=MBI_SSL, siteId=\"ssl.live-tst.net\"")]

```

Comment:
Can we have a test for each of the possible syntaxes in this spec? 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/WWW-Authenticate

Probably ok if we don't parse the single token. But let's make sure we don't crash.